app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
+            <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+            <!--<a href="/releases/#{{ rule.mapping }}"><b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b></a>-->
+            <a href="" ng-show="rule.mapping" ng-click="openReleaseDataModal(rule.mapping)">(view)</a>
+            <a href="/releases/#{{ rule.mapping }}">(edit)</a>
```

Comment:
We open the release blob in view-only mode. There is no in-line edit mode in the UI (yet). It does the same thing that "View Data" on the releases page does. Maybe I should change the text of the link to match?